client/utils/account-fees.tsx

```diff
@@ -25,8 +25,10 @@ const countryFeeStripeDocsSectionNumbers: Record< string, string > = {
 	AU: 'australia',
 	AT: 'austria',
 	BE: 'belgium',
+	BG: 'bulgaria',
 	CA: 'canada',
 	CY: 'cyprus',
+	CZ: 'czechia',
```

Comment:
Can we double check if it's czechia or czech-republic?

Reply:
Double-checked with @aheckler, this should be `czech-republic`. Fixed in https://github.com/Automattic/woocommerce-payments/pull/6215/commits/e4fe789d64faf59a175f75de6dff1ffe424b4771